collectors/carbonblack/test/utils_test.js

```diff
@@ -115,5 +115,104 @@ describe('Unit Tests', function () {
         });
     });
 });
+describe('Get API Logs (GET) with Error', function () {
```

Comment:
how much percentange does this code covered pls post it

Reply:
<img width="920" alt="image" src="https://user-images.githubusercontent.com/98916725/219318526-059bafbf-02dc-4ca1-93ae-f8d300078f6e.png">

added this to cover the .catch part and after adding this 
statement coverage increases from 95.31% to 96.87%
branch coverage is same 80%
function coverage increases from 88.88% to 100% and 
lines coverage increases from 95.31% to 96.87%